app-frontend/src/app/components/colorCorrectScenes/colorCorrectScenes.controller.js

```diff
@@ -1,3 +1,22 @@
+let lng2Tile = function (lng, zoom) {
+    return Math.floor((lng + 180) / 360 * Math.pow(2, zoom + 2));
+};
```

Comment:
I think some brief docstrings here explaining this functions/params would be good and what the return values represent